recipes.js

```diff
@@ -9,3 +9,76 @@ mongoose.connect('mongodb://localhost/recipeApp')
     console.error('Error connecting to mongo', err)
   });
 
+const recipeSchema = new Schema ({
+  title: {
+    type: String,
+    required: true
+  },
+  level: {
+    type: String,
+    enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef']
+  },
+  ingredients: [ String ],
```

Comment:
Here, you are saying that it is going to be an array of Strings, you can also do it like that : 
  ingredients:  { type: Array },